modules/Sources/Features/SendFlow/SendFlowView.swift

```diff
@@ -124,7 +125,7 @@ public struct SendFlowView: View {
                                 Text(L10n.Send.review.uppercased())
                             }
                             .zcashStyle()
-                            .disabled(!viewStore.isValidForm || viewStore.isSending)
+                            //.disabled(!viewStore.isValidForm || viewStore.isSending)
```

Comment:
Either add a TODO and file an issue, or delete the commented-out code.

Reply:
It was temporary, disable reactivated for the final PR